### ISSUE TITLE
Select throws warning on console dev tools

### ISSUE THREAD

**Issue by cristiandouce** [auth0/cosmos#1129]
**Describe the bug**
Using the Select component I'm getting this error on developer console tools:

```
Warning: Use the `defaultValue` or `value` props on <select> instead of setting `selected` on <option>.
```

**Expected behavior**
I'd expect no errors in the console.

**Screenshots**
![image](https://user-images.githubusercontent.com/367429/48916140-e967d500-ee5f-11e8-8ec1-a459867ac7c0.png)

**Cosmos version:**
`0.7.2`

**Additional context**
Usage:

```js
<Select
  value={selectedItem}
  options={items}
  onChange={onChangeHandler}
/>
```

where items has the following format:

```js
{
  text: 'Phone Number',
  value: 'phone_number'
}
```

**Comment by francocorreasosa** [comment]
@cristiandouce Is this still happening? I cannot reproduce it in `cosmos@latest`

**Comment by cristiandouce** [comment]
Seems fixed in latest `0.9.3` 🚀

**Comment by francocorreasosa** [comment]
Nice!
